apps/docs/src/pages/en/features/commander.md

```diff
@@ -80,6 +80,21 @@ For more details on everything that is possible with options, take a look at [`c
 
 You can also make an option completely required, like an argument, by setting `required: true` in the metadata for the option.
 
+### Variadic Options
+
+Options allow for variadic inputs but you will need to provide an option parser that accumulates each option.
```

Comment:
```suggestion
Options also allow for variadic inputs but you will need to provide an option parser that accumulates each option.
```

Reply:
Very minor change, everything else in this looks great! Thanks for the PR :smile: